docs/localization.rst

```diff
@@ -29,28 +29,53 @@ Getting the Localizations
 Localizations are found in this repository under the ``locale`` folder.
 
 The gettext portable object (.po) files need to be compiled into the gettext
-machine object (.mo) files before translations will appear. This is performed
-during vagrant provisioning but if you need to update them at any time you can
-compile the files via the following command within the vagrant environment::
+machine object (.mo) files before translations will appear. Though its not
+performed automatically in docker environment , it is performed during vagrant
+provisioning. If you need to update them at any time you can compile the files
+via the following command according to your environment:
+
+**In Docker**::
```

Comment:
Instead of giving versions of both commands, you should add a paragraph at the top that says something like:

Unless otherwise noted, run all the commands in this document inside the environment. For Vagrant, enter the environment with ``vagrant ssh``. For Docker, enter the environment with ``docker exec -it kuma_web_1``.

This is the pattern used on the [development](https://kuma.readthedocs.io/en/latest/development.html) document, and it worked pretty well.